docs/user-guide/how-to.md

```diff
@@ -388,6 +396,12 @@ To set `my-secret` not to propagate to any namespace, you can use:
 kubectl annotate secret my-secret -n parent propagate.hnc.x-k8s.io/none=true
 ```
 
+To set `my-secret` to propagate to any namespace, you can use:
```

Comment:
Suggestion: To set `my-secret` to propagate to any namespace when the mode is `AllowPropagate`, you can use: